### PR TITLE
Sign UWP artifacts

### PR DESCRIPTION
**Pull request by chcosta**
These are mostly @nattress 's changes, with the exception that I moved this section...

```
      <FilesToSign Include="$(UWPOutputDir)*.appx"> 
        <Authenticode>$(CertificateId)</Authenticode> 
      </FilesToSign> 
```

The appx files are produced during the "Build NuGet Packages" step, so they wouldn't be available for signing during the binaries phase.

FYI @joshfree @zamont @nattress

### CODE REVIEW

File: tools-local/tasks/ValidateBinInspectResults.cs [hunk 1]
```diff
@@ -52,8 +52,7 @@ public override bool Execute()
             {
                 foreach(var baselineFile in BaselineFiles)
                 {
-                    Regex ignoreRegEx = new Regex(baselineFile.ItemSpec);
-                    IEnumerable<XElement> baselineExcluded = errorRows.Where(f => ignoreRegEx.IsMatch(f.Descendants(s_NameElement).First().Value));
+                    IEnumerable<XElement> baselineExcluded = errorRows.Where(f => Regex.IsMatch(f.Descendants(s_NameElement).First().Value), baselineFile.ItemSpec, RegexOptions.IgnoreCase);
```
Comment (zamont) [inline tools-local/tasks/ValidateBinInspectResults.cs]: The Regex.IsMatch call seems to be causing the build failures

Comment (zamont) [inline tools-local/tasks/ValidateBinInspectResults.cs]: Based on this StackOverflow [link](https://stackoverflow.com/questions/2439965/case-insensitive-regex-without-using-regexoptions-enumeration), you might be able to keep the old code, except make the first line:

```
Regex ignoreRegEx = new Regex($"(?i){baselineFile.ItemSpec}");
```

Comment (chcosta) [inline tools-local/tasks/ValidateBinInspectResults.cs]: Thanks, this is what I get for trying to write code from my phone when I'm half asleep.  I'll take a look.

**Comment by chcosta** [comment]
Test windows_nt arm Debug build

**Pull request merged**